telegram/_message.py

```diff
@@ -255,94 +259,128 @@ class Message(TelegramObject):
+            programming languages may have difficulty/silent defects in interpreting it. But it is
+            smaller than 52 bits, so a signed 64 bit integer or double-precision float type are
+            safe for storing this identifier.
         migrate_from_chat_id (:obj:`int`): Optional. The supergroup has been migrated from a group
-            with the specified identifier.
-        pinned_message (:class:`telegram.Message`): Optional. Specified message was pinned.
-        invoice (:class:`telegram.Invoice`): Optional. Information about the invoice.
-        successful_payment (:class:`telegram.SuccessfulPayment`): Optional. Information about the
-            payment.
+            with the specified identifier. This number may be greater than 32 bits and some
+            programming languages may have difficulty/silent defects in interpreting it. But it is
+            smaller than 52 bits, so a signed 64 bit integer or double-precision float type are
+            safe for storing this identifier.
+        pinned_message (:class:`telegram.Message`): Optional. Specified message was pinned. Note
+            that the Message object in this field will not contain further 
+            :attr:`reply_to_message` fields even if it is itself a reply.
+        invoice (:class:`telegram.Invoice`): Optional. Message is an invoice for a payment,
+            information about the invoice.
+        successful_payment (:class:`telegram.SuccessfulPayment`): Optional. Message is a service
+            message about a successful payment, information about the payment.
         connected_website (:obj:`str`): Optional. The domain name of the website on which the user
             has logged in.
-        forward_signature (:obj:`str`): Optional. Signature of the post author for messages
-            forwarded from channels.
-        forward_sender_name	(:obj:`str`): Optional. Sender's name for messages forwarded from users
-            who disallow adding a link to their account in forwarded messages.
+        forward_signature (:obj:`str`): Optional. For messages forwarded from channels, signature
+            of the post author if present.
+        forward_sender_name	(:obj:`str`): Optional. Sender's name for messages forwarded from
+            users who disallow adding a link to their account in forwarded messages.
         author_signature (:obj:`str`): Optional. Signature of the post author for messages in
             channels, or the custom title of an anonymous group administrator.
         passport_data (:class:`telegram.PassportData`): Optional. Telegram Passport data.
         poll (:class:`telegram.Poll`): Optional. Message is a native poll,
             information about the poll.
-        dice (:class:`telegram.Dice`): Optional. Message is a dice.
-        via_bot (:class:`telegram.User`): Optional. Bot through which the message was sent.
+        dice (:class:`telegram.Dice`): Optional. Message is a dice with random value from 1 to 6.
```

Comment:
```suggestion
        dice (:class:`telegram.Dice`): Optional. Message is a dice with random value.
```
same for Args section

Reply:
done